crates/core/tedge_agent/src/restart_manager/actor.rs

```diff
@@ -116,7 +116,7 @@ impl RestartManagerActor {
     }
 
     async fn process_pending_restart_operation(&mut self) -> Option<RestartCommand> {
-        match self.state_repository.load().await {
+        match self.load_state_repository().await {
```

Comment:
What's the motivation for this change?

I saw the command is *sanitized* by `load_state_repository()`, but wonder why this is required. Touching a critical piece of code sounds inadequate for this log-related PR.

Reply:
Obsolete, as we switched to a different solution.